config/initializers/assets.rb

```diff
@@ -1,15 +1,13 @@
 # frozen_string_literal: true
 
+# govuk-frontend related assets
+Rails.application.config.assets.paths << Rails.root.join('node_modules/govuk-frontend/dist/govuk/assets/images')
+Rails.application.config.assets.paths << Rails.root.join('node_modules/govuk-frontend/dist/govuk/assets/fonts')
+
+# Add Yarn node_modules folder to the asset load path.
```

Comment:
Are these still needed? It looks like we add all the `node_modules` path to assets in line 8 🤔

Reply:
Yes, I believe we still need them, I get an error if I remove them. I don't think rails searches through the directories in the node modules for performance reasons and potential directory name conflicts.

<img width="1791" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/01f39a95-69c8-4ad0-90cd-16a5eed6ced8">

Reply:
@nulty and I took a look at this together and did this, was this what you had in mind @dcyoung-dev?

Reply:
I reverted these changes as it was causing more issues than it was worth.